Cargo.toml

```diff
@@ -57,3 +57,6 @@ exclude = []
 [profile.perf]
 inherits = "release"
 debug = true
+
+[workspace.dependencies]
+zksync_protobuf = { version = "0.1.0", git = "https://github.com/matter-labs/era-consensus.git", rev = "9b300371512a4e443d086ecd19c1877719c15646" }
```

Comment:
I'm afraid this will break the prover workspace, we'll need to fix that first :( 
(prover workspace was the reason why I had to rollback all my attempts to switch to workspace deps in the past

Reply:
ack, removed workspace.dependencies